commit c97b22d57e9a12827ff0ca0a611121c321e1412a (refs/changes/13/3213/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-01-15T11:17:51-08:00 (1 year, 9 months ago)
    
    TRITON-49 vms.changefeed.test.js throwing error makes whole VMAPI tests suite abort

diff --git a/test/vms.changefeed.test.js b/test/vms.changefeed.test.js
index dbf688c..179c6fc 100644
--- a/test/vms.changefeed.test.js
+++ b/test/vms.changefeed.test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 var assert = require('assert-plus');
@@ -97,7 +97,7 @@ function waitForValue(url, key, value, callback) {
             times++;
 
             if (times === TIMEOUT) {
-                throw new Error('Timeout waiting on ' + url);
+                callback(new Error('Timeout waiting on ' + url));
             } else {
                 setTimeout(function () {
                     waitForValue(url, key, value, callback);
